src/MSBuildLocator/Microsoft.Build.Locator.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks>net46;netcoreapp2.1</TargetFrameworks>
+    <TargetFrameworks>net46;netcoreapp3.1</TargetFrameworks>
```

Comment:
I think this needs a version bump; it's technically semver major but I don't know if we need to go that far but it seems weird to do it in a patch-level change (default for no `version.json` changes).